apps/agent-provisioning/AFJ/scripts/docker_start_agent.sh

```
@@ -17,6 +17,13 @@ PROTOCOL=${12}
 TENANT=${13}
 AFJ_VERSION=${14}
 INDY_LEDGER=${15}
+INBOUND_ENDPOINT=${16}
```

Shellcheck (0.10.0)

[warning] INBOUND_ENDPOINT appears unused. Verify use (or export if used externally).

(SC2034)